app/Console/Commands/SendIntegrationActivationReminderEmailCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use App\Domain\Integrations\IntegrationType;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Mails\MailJet\MailjetConfig;
+use App\Mails\SendIntegrationActivationReminderEmail;
+use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+
+final class SendIntegrationActivationReminderEmailCommand extends Command
+{
+    protected $signature = 'cronjob:send-activation-reminder-email {--force : Skip confirmation prompt}';
+
+    protected $description = 'Send activation reminder email';
+
+    public function __construct(
+        private readonly IntegrationRepository $integrationRepository,
+        private readonly SendIntegrationActivationReminderEmail $sendIntegrationActivationReminderEmail,
+    ) {
+        parent::__construct();
+    }
+
+    public function handle(): int
+    {
+        if (!config(MailjetConfig::TRANSACTIONAL_EMAILS_ENABLED)) {
+            $this->output->writeln('Email feature flag is disabled - mails not sent');
+            return self::FAILURE;
+        }
+
+        $integrationTypeConfig = [
+            IntegrationType::SearchApi->value => 6,
```

Comment:
What are the 6 and 3? If those are the IDs of the Mailjet templates, then it may be worth putting them in an enum.

Reply:
Those are after how many months we sent an email. 
I made constants out of them.